.github/workflows/continuous-integration-workflow.yml

```diff
@@ -294,6 +296,20 @@ jobs:
           ./gradlew :spring-security-docs:antoraUpdateVersion
           git commit -am "Next development version"
           git push
+  perform_post_release:
+    name: Perform post-release
+    needs: [prerequisites, deploy_artifacts, deploy_docs, deploy_schema]
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    if: ${{ endsWith(needs.prerequisites.outputs.project_version, '-SNAPSHOT') }}
+    env:
+      TOKEN: ${{ github.token }}
+      VERSION: ${{ needs.prerequisites.outputs.project_version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: spring-io/spring-gradle-build-action@v1
+      - name: Schedule next release (if not already scheduled)
+        run: ./gradlew scheduleNextRelease -PnextVersion=$VERSION -PgitHubAccessToken=$TOKEN
```

Comment:
Is nextVersion an accurate name here? To me it appears that this is the current version because the release has been performed. The scheduleNextRelease task refers to it as version so perhaps that is a better argument? 

i'm also curious if Gradle really needs to read the version from the prerequisites.outputs. Could Gradle just access the project.version instead?

Reply:
I think I see and agree now. nextVersion is in the context of what is being scheduled and that had not been done yet even though the release has already been done. Thanks!